barretenberg/cpp/src/barretenberg/circuit_checker/mega_circuit_builder.test.cpp

```diff
@@ -13,21 +13,20 @@ namespace bb {
 
 TEST(MegaCircuitBuilder, CopyConstructor)
 {
-    MegaCircuitBuilder circuit_constructor = MegaCircuitBuilder();
+    MegaCircuitBuilder builder = MegaCircuitBuilder();
```

Comment:
I found some remains of the term circuit constructor here and in the ultra circuit builder tests and addressed them as well